presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -504,6 +505,11 @@ public SystemSessionProperties(
                         "Skip redundant sort operations",
                         featuresConfig.isSkipRedundantSort(),
                         false),
+                booleanProperty(
```

Comment:
Do we need this property? Shouldn't improved property derivation help always?

Reply:
It's a fail-safe in case we missed something and there's a bug. We'll remove it in a few releases

Reply:
`@Slated`